internal/rbd/replicationcontrollerserver.go

```diff
@@ -445,20 +436,16 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 	}
 	ready := false
 	state := imageMirroringState(mirrorStatus.State)
-	if state == upAndStopped || state == upAndReplaying {
-		// Make sure the peer site image state is up and stopped
-		ready = true
-		for _, s := range mirrorStatus.PeerSites {
-			if imageMirroringState(s.State) != upAndStopped {
-				util.UsefulLog(ctx, "peer site name=%s, mirroring state=%s, description=%s and lastUpdate=%s", s.SiteName, s.State, s.Description, s.LastUpdate)
-				ready = false
-			}
-		}
-	}
-
-	// when the images are demoted on both clusters and user requests for the
-	// resync of the image, the image mirror state will be unknown state in
-	// both clusters.
+	//  To recover from split brain (up+error) state the image need to be
```

Comment:
If the split brian scenario is `up+error` dont we better make use of a mirror deamon state check here ? I only see the error return and no validation of `mirror` deamon state, so trying to clarify @Madhu-1 

```
// resync only if the image is in error state
	if strings.Contains(mirrorStatus.State, string(errorState)) {
```

Reply:
up is the daemon state and error is the image state.

Reply:
okay, I think we don't need to worry about the daemon state here. we will issue the resync command from our side. if the daemon is not running the image will not get recovered.

Reply:
ok..